Aliases/br

```diff
@@ -0,0 +1 @@
+../Formula/brooklyn-client.rb
```

Comment:
@MikeMcQuaid going to go with `apache-brooklyn-cli`, just holding off on the commit for a bit to allow additional feedback on the dev list.

Do you want me to squash the commits when I'm done?